rest/doc_api_test.go

```diff
@@ -126,11 +126,11 @@ func TestDocumentNumbers(t *testing.T) {
 
 			// Check channel assignment
 			getRawResponse := rt.SendAdminRequest("GET", fmt.Sprintf("/{{.keyspace}}/_raw/%s?redact=false", test.name), "")
-			var rawResponse RawResponse
+			var rawResponse RawDocResponse
 			require.NoError(ts, base.JSONUnmarshal(getRawResponse.Body.Bytes(), &rawResponse))
 			log.Printf("raw response: %s", getRawResponse.Body.Bytes())
-			assert.Equal(ts, 1, len(rawResponse.Sync.Channels))
-			assert.True(ts, HasActiveChannel(rawResponse.Sync.Channels, test.expectedFormatChannel), fmt.Sprintf("Expected channel %s was not found in document channels (%s)", test.expectedFormatChannel, test.name))
+			assert.Equal(ts, 1, len(rawResponse.Xattrs.Sync.Channels))
+			assert.True(ts, HasActiveChannel(rawResponse.Xattrs.Sync.Channels, test.expectedFormatChannel), fmt.Sprintf("Expected channel %s was not found in document channels (%s)", test.expectedFormatChannel, test.name))
```

Comment:
I've thought about adding an assertion to `base/util_testing.go`:

`RequireMapKeysEqual` with the implementation:

`require.ElementsEqual(t, expectedValues, slices.Collect(maps.Keys(actual)), "Actual map: %#+v", actual)`

then you would know what the actual values where whereas `HasActiveChannel` doesn't tell you what the keys are. The difference in `HasActiveChannel` is that it would tell you if the `* ChannelRemoval` was null.

Reply:
`assert.Contains` works well instead. This is just old code.